### PR TITLE
Add angela97lin as maintainer

### PR DESCRIPTION
**Pull request by angela97lin**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ParthivNaresh** [comment]
@conda-forge-admin, please rerender

**Review by freddyaboulton — APPROVED** [review]
Looks good!

**Comment by angela97lin** [comment]
@freddyaboulton @dsherry I can't merge this (no write-access) 😭 If you guys could, that'd be amazing!

**Pull request merged**
